package.json

```diff
@@ -20,9 +22,19 @@
     "react-dom": "17.0.1"
   },
   "devDependencies": {
+    "@types/node": "^14.14.22",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
```

Comment:
See: https://github.com/vercel/next.js/tree/canary/examples/with-typescript#notes and https://nextjs.org/docs/basic-features/typescript